src/test/java/org/semux/crypto/NativeTest.java

```diff
@@ -54,12 +54,12 @@ public static void teardown() {
 
     @Test(expected = CryptoException.class)
     public void testH256Null() {
-        Native.h256(null);
+        Native.h256(null, null);
```

Comment:
expect the NativeTest to be updated to actually test the salted implementation for native matches the non-native one.